src/OmniSharp/Models/SignatureHelp.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace OmniSharp.Models
+{
+    public class SignatureHelp
+    {
+        public IEnumerable<SignatureHelpItem> Signatures { get; set; }
+
+        public int ActiveSignature { get; set; }
```

Comment:
Maybe this might be a little easier to work with if we had `bool Active` on SignatureHelpItem?